### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by dolph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,11 +92,11 @@ To access the machine learning service programmatically, you need to copy in you
 * Open the `Service Credentials` tab and view the credentials.
 * Copy your credentials and replace the `**URL**`, `**USERNAME**`, `**PASSWORD**` and `**INSTANCE_ID**` placeholders in the notebook and run the cell.
 
-## 6. Deploy the model to the Cloud and run the product recommendation PixieApp
+### 6. Deploy the model to the Cloud and run the product recommendation PixieApp
```
Comment (ptitzler) [inline README.md]: Why the demotion to H4 from H3?

File: README.md [hunk 2]
```diff
@@ -45,14 +45,14 @@ Follow these steps to setup and run this code pattern. The steps are described i
 
 1. [Sign up for Watson Studio](#1-sign-up-for-watson-studio)
 1. [Create a project and add services](#2-create-a-project-and-add-services)
-1. [Create a notebook](#3-create-the-notebook)
-1. [Load customer data in the notebook](#4-load-customer-data)
-1. [Add a Watson Machine Learning service](#5-add-watson-machine-learning)
-1. [Build and run the recommendation PixieApp](#6-build-the-pixieapp)
+1. [Create a notebook](#3-create-a-notebook)
+1. [Load customer data in the notebook](#4-load-customer-data-in-the-notebook)
+1. [Add the Watson Machine Learning credentials to the notebook](#5-add-watson-machine-learning-credentials-to-the-notebook)
```
Comment (ptitzler) [inline README.md]: It appears the HREF is broken. It should be `#5-add-the-watson-machine-learning-credentials-to-the-notebook`

**Review by ptitzler — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dolph) [reply README.md]: Good catch - I'm not sure what I was thinking. I'll revert.

**Review by ptitzler — APPROVED** [review]
LGTM

**Review by stevemar — APPROVED** [review]

**Pull request merged**